metadata.json

```
@@ -4,7 +4,8 @@
   "name": "Extensions Sync",
   "description": "Sync all extensions and their configurations across all gnome instances through gist",
   "shell-version": [
-    "3.32"
+    "3.32",
+    "3.34",
```

It seems to be a non valid JSON (`Failed to parse metadata.json: SyntaxError: JSON.parse: unexpected character at line 9 column 3 of the JSON data`)

```suggestion
    "3.34"
```